client/app/components/retire-service-modal/retire-service-modal-service.factory.js

```diff
@@ -62,10 +63,18 @@
       { value: 21, label: __('3 Weeks') },
       { value: 28, label: __('4 Weeks') },
     ];
+    vm.visibleOptions = [];
 
     activate();
 
     function activate() {
+      $scope.$watch('vm.modalData.resource.date', function(date) {
+        var daysBetween = moment(date).diff(moment(), 'days');
```

Comment:
No this is just an identifier for moment's `diff` method to indicate unit of measure:

http://momentjs.com/docs/#/displaying/difference/

It is not a string the user sees.